scripts/scil_visualize_seeds.py

```diff
@@ -46,7 +46,7 @@ def main():
         raise ValueError("Invalid input streamline file format " +
                          "(must be trk): {0}".format(args.tractogram_filename))
 
-    # Load files and data. TRKs can have 'same' as reference
+    # Load files and data. TRKs can have 'same' as refescenece
```

Comment:
Typo, right?

Reply:
If it is about the Mediterranean recluse spider, it is rufescens, not refescenece ;)

Reply:
Ha ! My regex might have been too permissive. I'll fix that asap.